### PR TITLE
Feature/calculator UI

### PR DESCRIPTION
**Pull request by shahadat3669**
### Math Magicians: Components

image

- [x] I develop a React component that will hold the core functionality: a calculator.
- [x] I use class-based components (All components should be class-based).
- [x] Once the Calculator is ready, import it and use it in the main component (App.js).
- [x] Test it in the browser; it should render without problems.
- [x] Delete all the boilerplate from CRA (text, images, styles).
- [x] Create a directory called components.
- [x] Inside components create a new Calculator.js file.
- [x] In Calculator.js, create a React component that matches this design:

#### Others:
- [x] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Make sure that you used correctly [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```
Comment (prolajumokeoni) [inline src/App.js]: - [x] Kindly make the App.js component class-based this is a requirement for this milestone 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @shahadat3669 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
- [x] Kindly delete all the boilerplate that got created by the CRA (texts, images, and styles) 👍🏿 

211594358-0536cb85-4293-4fe0-a528-3777642eff18

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (shahadat3669) [reply src/App.js]: @prolajumokeoni I added a logo and named it logo512.png. Do I have to delete it?

**Comment by shahadat3669** [comment]
### Required Changes ♻️

- [x] Make the App.js component class-based.
- [x] Delete all the boilerplate that got created by the CRA

_Note: logo512.png is uploaded by me to use in the readme file._
image

File: src/App.js (hunk 1 above)
Comment (Gambit142) [reply src/App.js]: @shahadat3669, if you have an image that you added to the project, please leave it. Boilerplate images/files/directories are those that came with the default application when you ran the Create React Application command. 👍

**Review by Gambit142 — APPROVED** [review]
Hi @shahadat3669 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- I love your understanding of having reusable components 💯  
- README.md file is very professional ✔️
- All components are _**class-based**_ ✔️ 
- All boilerplates have been deleted ✔️  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**